packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountsYouManageHomePage.tsx

```diff
@@ -37,6 +43,28 @@ export const AccountsYouManageHomePage = ({
   const [rejectManagedAccount, rejectResult] = useRemoveManager()
   const { toast } = useContext(ToastContext)
 
+  useEffect(() => {
+    if (managedAccounts == null) return
+    if (
+      pending != null &&
+      typeof pending === 'string' &&
+      pending.trim() !== '' &&
+      !isNaN(Number(pending))
+    ) {
+      const pendingUserId = Number(pending)
+      const pendingManagedAccount = managedAccounts.find(
+        (m) => m.grant.user_id === pendingUserId
+      )
+      if (!pendingManagedAccount) {
+        toast(messages.invalidInvitation)
+        return
+      }
+      if (pendingManagedAccount.grant.is_approved) {
+        toast(messages.alreadyAcceptedInvitation)
+      }
+    }
+  }, [toast, pending, managedAccounts])
```

Comment:
What happens when `managedAccounts` updates while we're still on this page and now you've accepted the invite?